src/web/jdv/server/kmz.py

```diff
@@ -164,7 +164,7 @@ def kml_from_task_packets(task_packets: Iterable[TaskPacket]):
     '''
 
 
-def write_file(task_packets: Iterable[TaskPacket], output_kmz_path: str):
+def writeTaskPacketsToKMZ(task_packets: Iterable[TaskPacket], output_kmz_path: str):
```

Comment:
Can we add the a comment before this function like the driftspeedtobinindex function  /***/ (This is python so the equivalent #)